# SolveRequest
trajectory_msgs/JointTrajectory pred_traj
float64[] joint_start
float64[] start_vel
float64[] object_set
float64[] weights
trajectory_msgs/JointTrajectory nom_traj
float64[] end_goal
---
# SolveResponse
trajectory_msgs/JointTrajectory solve_traj